README.md

```diff
@@ -1 +1,41 @@
-# Hello-World
\ No newline at end of file
+Hello Microverse
+Display Hello World with red color.
+
+Built With
+- HTML
+- CSS
+
+Getting Started
+To get a local copy up and running:
+
+1. Clone this repository or download the zip folder:
+
+**``git clone https://github.com/HtetNaing0814/Hello-World.git``**
+
+1. Navigate to the location of the folder in your machine:
+
+**``you@your-Pc-name:~$ cd <folder>``**
+
+To get a local copy up and running follow these simple example steps.
+
+Authors
+👤 - Github: [@HtetNaing0814](https://github.com/HtetNaing0814)
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issue page](https://github.com/HtetNaing0814/Hello-World/issues).
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+[Microverse](https://bit.ly/MicroverseTN)
+
+📝 License
+This project is [MIT](https://github.com/microverseinc/readme-template/blob/master/MIT.md
+) licensed # as a link
+
+This project is [MIT](./MIT.md) licensed # add a file by adding the `MIT.md` file in the root folder of your `Hello Microverse Project` 
+
+This project is MIT licensed.
```

Comment:
- [ ] [_Optional_] Please ensure you use only 1 method suggested by the Previous Reviewer, right now you have 3 statements saying that the ReadMe is MIT licensed.